src/measure/dump_polarizability.cu

```diff
@@ -0,0 +1,203 @@
+/*
+    Copyright 2017 Zheyong Fan, Ville Vierimaa, Mikko Ervasti, and Ari Harju
+    This file is part of GPUMD.
+    GPUMD is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    GPUMD is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details. You should have received a copy of the GNU General
+   Public License along with GPUMD.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/*-----------------------------------------------------------------------------------------------100
+Dump energy/force/virial with all loaded potentials at a given interval.
+--------------------------------------------------------------------------------------------------*/
+
+#include "dump_polarizability.cuh"
+#include "model/box.cuh"
+#include "model/read_xyz.cuh"
+#include "parse_utilities.cuh"
+#include "utilities/common.cuh"
+#include "utilities/error.cuh"
+#include "utilities/gpu_vector.cuh"
+#include "utilities/read_file.cuh"
+#include <iostream>
+#include <vector>
+
+static __global__ void sum_polarizability(
+  int N, const double* g_potential_per_atom, const double* g_virial_per_atom, double* g_pol)
```

Comment:
The summation here is no correct. You need to either use `atomicAdd`, or use the so-called binary reduction. You can check e.g., the function, `gpu_sum_tensor()` in `force.cu`.